components/contribution-flow/ContributionBlocker.js

```diff
@@ -70,7 +70,7 @@ export const getContributionBlocker = (loggedInUser, account, tier, shouldHaveTi
     return { reason: CONTRIBUTION_BLOCKER.TIER_MISSING, type: 'warning', showOtherWaysToContribute: true };
   } else if (tier && isTierExpired(tier)) {
     return { reason: CONTRIBUTION_BLOCKER.TIER_EXPIRED, type: 'warning', showOtherWaysToContribute: true };
-  } else if (account.settings.disableCustomContributions && !tier) {
+  } else if (account.settings.disableCustomContributions && account.settings.disableCryptoContributions && !tier) {
```

Comment:
This is not right, if someone sets `disableCustomContributions` to true then `/donate` should be blocked, regardless of what is set in `disableCryptoContributions`.

Reply:
@Betree : Ah, that's a good point. I've reverted this condition but rather handled it in the `contribution-flow.js` itself; 

https://github.com/opencollective/opencollective-frontend/blob/7e91fcbf294d473f62275637351259b41dc4d8c0/pages/contribution-flow.js#L146

Let me know if you see any issues with that. My thinking is we should enable the crypto payment method regardless of contribution blocker conditions such as no-payment-method, because crypto flow doesn't use any payment methods and such. 🤔

Reply:
@Betree : I've refactored this based on the discussion on Slack; https://opencollective.slack.com/archives/C0RMV6F8C/p1631275468072800. Thanks for pointing it out. 🙂